compiler/luci-interpreter/src/loader/KernelBuilder.cpp

```diff
@@ -440,6 +441,34 @@ std::unique_ptr<Kernel> KernelBuilder::visit(const luci::CircleSplit *node)
   return std::make_unique<kernels::Split>(axis, input, std::move(outputs));
 }
 
+std::unique_ptr<Kernel> KernelBuilder::visit(const luci::CircleStridedSlice *node)
+{
+  assert(node->arity() == 4);
+
+  if (dynamic_cast<const luci::CircleConst *>(node->begin()) == nullptr)
+    throw std::runtime_error("Dynamic begin is not yet supported.");
+  if (dynamic_cast<const luci::CircleConst *>(node->end()) == nullptr)
+    throw std::runtime_error("Dynamic end is not yet supported.");
+  if (dynamic_cast<const luci::CircleConst *>(node->strides()) == nullptr)
+    throw std::runtime_error("Dynamic strides is not yet supported.");
```

Comment:
These checks are no longer needed, could you remove them in a followup PR? See #3051.

Reply:
Okay, I make up followup PR with below comment. Thanks!